src/main/java/com/eternalcode/combat/fight/controller/FightActionBlockerController.java

```diff
@@ -60,6 +61,24 @@ void onPlace(BlockPlaceEvent event) {
         }
     }
 
+    @EventHandler
+    public void onToggleGlide(EntityToggleGlideEvent event) {
+        if (!this.config.settings.shouldPreventElytraUsage) {
+            return;
+        }
+
+        Player player = event.getEntity() instanceof Player ? (Player) event.getEntity() : null;
+        UUID uniqueId = player != null ? player.getUniqueId() : null;
```

Comment:
Nie podoba mi się zastosowanie tego, ja bym po prostu zrobił return jeżeli entity nie jest Playerem, bo jeżeli `uniqueId` będzie nullem, a potem używasz tego do zapytania `isInCombat` to jest to raczej zachowanie niepożądane.